engine/src/main/java/org/terasology/rendering/world/WorldRendererLwjgl.java

```diff
@@ -193,31 +162,26 @@ public boolean pregenerateChunks() {
     }
 
     @Override
-    public void update(float delta) {
+    public void update(float deltaInMs) {
 
         PerformanceMonitor.startActivity("Update Tick");
```

Comment:
I double-checked the instances of PerformanceMonitor usage in WorldRendererLwjgl and RenderableWorld. I have not found any that framed a section of code with multiple return location, so no need for special handling so far.

The use of Closable objects to wrap sections of code is interesting though. I remember changing the GameEngine interface to extend AutoCloseable, which I seem to understand is similar. I wonder how difficult it would be to change the PerformanceMonitor to work with it. I'd rather not do it for the time being though. I have enough on my plate already.